app/views/sufia/admin/admin_sets/index.html.erb

```diff
@@ -27,6 +28,7 @@
                     <td><%= link_to admin_set.title.first, [sufia, :admin, admin_set] %></td>
                     <td><%= admin_set.create_date %></td>
                     <td><%= safe_join(admin_set.creator) %></td>
+                    <td><%= @presenter.total_items(admin_set) %></td>
```

Comment:
This works as long as `include Devise::Test::ControllerHelpers` is added to the context. Otherwise test errors with Devise could not find the `Warden::Proxy` instance on your request environment.